<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="8368" started-at="2018-01-26T22:37:26Z" finished-at="2018-01-26T22:37:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="8368" started-at="2018-01-26T22:37:26Z" finished-at="2018-01-26T22:37:34Z">
      <class name="NewTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:NewTest@42d80b78]" name="beforeTest" is-config="true" duration-ms="2601" started-at="2018-01-26T19:37:26Z" finished-at="2018-01-26T19:37:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="App_Title_CelFar2()[pri:0, instance:NewTest@42d80b78]" name="App_Title_CelFar2" duration-ms="2065" started-at="2018-01-26T19:37:28Z" finished-at="2018-01-26T19:37:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- App_Title_CelFar2 -->
        <test-method status="FAIL" signature="Page_Title_CelFar()[pri:0, instance:NewTest@42d80b78]" name="Page_Title_CelFar" duration-ms="305" started-at="2018-01-26T19:37:30Z" finished-at="2018-01-26T19:37:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [CelFar] but found [Conversor CelFar]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [CelFar] but found [Conversor CelFar]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at NewTest.Page_Title_CelFar(NewTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Page_Title_CelFar -->
        <test-method status="PASS" signature="conL_1234567()[pri:0, instance:NewTest@42d80b78]" name="conL_1234567" duration-ms="465" started-at="2018-01-26T19:37:31Z" finished-at="2018-01-26T19:37:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- conL_1234567 -->
        <test-method status="PASS" signature="con_0()[pri:0, instance:NewTest@42d80b78]" name="con_0" duration-ms="431" started-at="2018-01-26T19:37:31Z" finished-at="2018-01-26T19:37:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- con_0 -->
        <test-method status="PASS" signature="con_10()[pri:0, instance:NewTest@42d80b78]" name="con_10" duration-ms="413" started-at="2018-01-26T19:37:32Z" finished-at="2018-01-26T19:37:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- con_10 -->
        <test-method status="PASS" signature="con_20()[pri:0, instance:NewTest@42d80b78]" name="con_20" duration-ms="464" started-at="2018-01-26T19:37:32Z" finished-at="2018-01-26T19:37:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- con_20 -->
        <test-method status="PASS" signature="con_m300()[pri:0, instance:NewTest@42d80b78]" name="con_m300" duration-ms="500" started-at="2018-01-26T19:37:33Z" finished-at="2018-01-26T19:37:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- con_m300 -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:NewTest@42d80b78]" name="afterTest" is-config="true" duration-ms="1117" started-at="2018-01-26T19:37:33Z" finished-at="2018-01-26T19:37:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
